specs/StreetLighting/StreetlightControlCabinet/schema.json

```diff
@@ -0,0 +1,285 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/specs/StreetLighting/StreetlightControlCabinet/schema.json",
+  "title": "FIWARE - Street Lighting / Streetlight Control Cabinet",
+  "description": "A Streetlight control cabinet",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons" },
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/PhysicalObject-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "StreetlightControlCabinet"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "areaServed": {
+          "type": "string",
+          "description": "Higher level area to which this streetlight belongs to. It can be used to group streetlights per responsible, district, neighbourhood, etc."
+        },
+        "serialNumber": {
+          "type": "string",
+          "description": "Serial number of the control cabinet."
+        },
+        "refStreetlightGroup": {
+          "type": "array",
+          "items": {
+              "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType" },
+          "description": "Streetlight group(s) controlled."
+        },
+        "brandName": {
+          "type": "string",
+          "description": "Name of the cabinet's brand."
+        },
+        "modelName": {
+          "type": "string",
+          "description": "Name of the cabinet's model."
+        },
+        "manufacturerName": {
+          "type": "string",
+          "description": "Name of the cabinet's manufacturer."
+        },
+        "cupboardMadeOf": {
+          "type": "string",
+          "enum": [
+            "plastic",
+            "metal",
+            "concrete",
+            "other"
+          ],
+          "description": "Material the cabinet's cupboard is made of."
+        },
+        "features": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "astronomicalClock",
+              "individualControl"
+            ]
+          },
+          "minItems": 0,
```

Comment:
this should be one, otherwise, you would be saying that no feature is available, isn't it?